packages/flutter_tools/lib/src/web/devfs_config.dart

```diff
@@ -189,28 +189,51 @@ WebDevServerConfig:
 /// Represents the [HttpsConfig] for the web dev server
 @immutable
 class HttpsConfig {
-  const HttpsConfig({this.certPath, this.certKeyPath});
-
+  const HttpsConfig({required this.certPath, required this.certKeyPath});
   factory HttpsConfig.fromYaml(YamlMap yaml) {
     final String? certPath = _validateType<String>(value: yaml[_kCertPath], fieldName: _kCertPath);
+    if (certPath == null) {
+      throw ArgumentError.value(yaml, 'yaml', '"$_kCertPath" must be defined');
```

Comment:
These throws should have broken some test but it seems they didn't. Is there a test for this part of the tool that can assert that certPath and certKeyPath are being handled correctly?

(I guess `fromYaml` is used to load this from the config file only?)

Reply:
I don't think there are tests here. Which is...sad...